source/navitiacommon/navitiacommon/models/equipments_providers.py

```diff
@@ -38,7 +38,7 @@
 
 class EquipmentsProvider(db.Model, TimestampMixin):  # type: ignore
     id = db.Column(db.Text, primary_key=True)
-    instances = db.Column(ARRAY(db.Text), unique=False, nullable=False)
+    instances = db.Column(ARRAY(db.Text), unique=False, nullable=True)
```

Comment:
Not sure it actually make sense to have Null instance name in the database? 
What was the use case behind this ?

Reply:
The use is that this column is useless and we want to remove it. To avoid breaking anything during the next deployment, where 'instances' won't be set, the db has to accept that this field can be empty.

Reply:
You have to leave the column in base, but remove it from the model (so: remove everything but the column in db).
Otherwise, it will break when we remove it from the base (old instance still trying to read it from base, event if it's doing nothing with it).